pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -828,3 +847,134 @@ func getSecurityProfileCiphers(profile *configv1.TLSSecurityProfile) (string, []
 	// need to remap all Ciphers to their respective IANA names used by Go
 	return string(profileSpec.MinTLSVersion), crypto.OpenSSLToIANACipherSuites(profileSpec.Ciphers)
 }
+
+// RunKubeletBootstrap generates MachineConfig objects for mcpPools that would have been generated by syncKubeletConfig
+// nolint:gocyclo
+func RunKubeletBootstrap(templateDir string, kubeletConfig *mcfgv1.KubeletConfig, controllerConfig *mcfgv1.ControllerConfig, mcpPools []*mcfgv1.MachineConfigPool) ([]*mcfgv1.MachineConfig, error) {
```

Comment:
would it make it easier to move this stuff to a kubeletbootstrap.go file or somesuch? it's awfully long

Reply:
yes, I think this function should reuse the other function this is derived from.

Reply:
@QiWang19 I know the func was removed but did you also reuse other func as Ryan suggested? Can't remember looking at it.